runtime/onert/core/include/ir/Model.h

```diff
@@ -175,8 +175,26 @@ class Model
 
 private:
   std::shared_ptr<backend::custom::IKernelBuilder> _kernel_builder;
-};
 
+public:
+  void add_metadata(const std::string &name, std::shared_ptr<const ir::Data> data)
+  {
+    _metadatas.emplace(name, data);
+  }
+
+  bool is_metadata_exist(const std::string &name) const
```

Comment:
What about `has_metadata`?

```suggestion
  bool has_metadata(const std::string &name) const
```

Reply:
How about?
```suggestion
  bool exists_metadata(const std::string &name) const
```

Reply:
I am okay both `has_metadata()` and `exists_metadata()`. I also thought `exists_metadata` as another candidate.

Reply:
thanks for the suggestion! 
updated.